src/gui/XournalView.cpp

```diff
@@ -278,24 +278,24 @@ bool XournalView::onKeyPressEvent(GdkEventKey* event)
 
 	if (event->keyval == GDK_KEY_Left)
 	{
-		if (control->getSettings()->isPresentationMode())
-		{
+		//if (control->getSettings()->isPresentationMode())
```

Comment:
Why comment out?

Reply:
It was functional to me, for skipping to the next/prev slide with just the right/left arrow key, without having to be in presentation mode. I guess the best thing to to would be to make it customizable

Reply:
I didn't ever use the presentation mode, therefore I'm asking...